sw/device/silicon_creator/lib/manifest.h

```diff
@@ -18,6 +18,59 @@
 extern "C" {
 #endif  // __cplusplus
 
+/**
+ * Usage constraints.
+ *
+ * This struct is used to constrain a boot stage image to a set of devices based
+ * on their device IDs, creator and/or owner manufacturing states, and life
+ * cycle states. Bits of `selector_bits` determine which fields (or individual
+ * words of a field as in the case of `device_id`) must be read from the
+ * hardware during verification. Unselected fields must be set to
+ * `MANIFEST_USAGE_CONSTRAINT_UNSELECTED_WORD_VAL` to be able to generate a
+ * consistent value during verification.
+ */
+typedef struct manifest_usage_constraints {
+  /**
+   * Usage constraint selector bits.
```

Comment:
We should probably have a table summarising the bit -> constraint mappings here.

Reply:
I agree, done.